docs/install/docker/windows-docker-ce.md

```diff
@@ -0,0 +1,144 @@
+---
+navTitle: Docker Engine on Windows
+meta:
+   description: Install Docker Engine (Docker CE) on Windows.
+---
+
+# How to Install Docker Engine (Docker CE) on Windows using WSL2
+
+## Disclaimer 
+
+Docker Containers are a Linux technology, and the setup described here using Docker on Windows via WSL2 should not be considered production-ready. It is more suitable for development, testing, or learning purposes.
+
+## Prerequisites
+
+Ensure your system meets following requirements:
+
+- Windows Server 2019 (version 1709) and later
+- Windows 10 version 2004 and higher (Build 19041 and higher)
+- Windows 11
+
+## Step 1: Install Windows Subsystem for Linux 
+
+Open PowerShell as an administrator and run the following commands:
+
+1. Install WSL and default Linux distribution (Ubuntu) using the following command:
+   ```powershell
+   wsl --install
+   ```
+
+2. Reboot your system to apply changes
+   ```powershell
+   shutdown -r -t 5
+   ```
+
+   After the reboot, WSL will automatically start installing Ubuntu Linux distribution.
+   You will be prompted to create a new UNIX account. Follow the instructions to create a new user account and set a password.
+   ![wsl-user](../images/wsl-unix-user.png)
+
+   Once completed, you will be dropped into a new Ubuntu shell.
+   ![wsl-complete](../images/wsl-install-complete.png)
```

Comment:
Hi @ppawlowski, could you please update the alt text to include more detailed information?